### DISCUSSION TITLE
MQTT Support

### DISCUSSION THREAD

**Discussion by x3style** [HandyHat/ha-hildebrandglow-dcc#234] [Ideas]
Hey, any chance you can implement MQTT support as well?

The other integration by unlobitto that was developed a long time ago for Glow DCC does not work anymore is archived and in read-only.

It would be really cool as the only remaining glow integration (i could find) to incorporate MQTT support for faster data which includes the near real time energy consumption metric.

**Comment by HandyHat** [comment]
If you have the display and CAD, you can use the local mqtt integration: https://github.com/megakid/ha_hildebrand_glow_ihd_mqtt

Unfortunately, as I do not have their hardware, I am unable to program for and test the usage of MQTT. If someone is interested in maintaining this, please reach out!

**Reply by x3style** [reply]
If you ask the guys at support@glowmarkt.com since they quote your integration as one of the few that work. I'm sure they won't mind sending you a free Glow DCC in order for you to support the DCC devices and enable them to sell more. AS far as I spoke with them they seem rather great people there.

**Reply by townsmcp** [reply]
@x3style also have a look at the very active thread on the HA community https://community.home-assistant.io/t/glow-hildebrand-display-local-mqtt-access-template-help/428638/256

**Reply by x3style** [reply]
> @x3style also have a look at the very active thread on the HA community https://community.home-assistant.io/t/glow-hildebrand-display-local-mqtt-access-template-help/428638/256

Thank you for the suggestion, I already have everything working as I have created my own manual sensors, however what I was looking for was that this becomes part of the HA core integrations available trough the official HA channels and that it includes all the ways one could integrate with Hildebrand glow Web API, MQTT, etc. I think this deserves to be a mature core integration of HA and I believe Hildebrand should drive it. So that you can easily configure the integration via a configuration menu.
